packages/ocaml-variants/ocaml-variants.4.07.0+trunk+afl/opam

```diff
@@ -12,7 +12,7 @@ flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
 build: [
   ["./configure" "-prefix" prefix "-afl-instrument"]
-    {os != "openbsd" & os != "freebsd" & os != "macos"}
+    {os != "openbsd" & os != "freebsd" & os != "darwin"}
```

Comment:
is this change correct ?

Reply:
hm, no, "macos" is the correct one. I'll check why this isn't fixed, thanks for notifying me.